unit-tests/run-unit-tests.py

```diff
@@ -33,7 +33,6 @@ def usage():
     print( '        -q, --quiet          Suppress output; rely on exit status (0=no failures)' )
     print( '        -s, --stdout         Do not redirect stdout to logs' )
     print( '        -r, --regex          Run all tests whose name matches the following regular expression' )
-    print( '        --skip-regex         Skip all tests whose name matches the following regular expression' )
```

Comment:
Looks like you are not aligned and you deleted my changes, please check

Reply:
Fixed